requirements.txt

```diff
@@ -1,2 +1,3 @@
 mock==2.0.0; python_version < "3.3"
 python-cas==1.4.0
+djangorestframework-simplejwt<=4.3.0
```

Comment:
😓 Oh boy. This is going to get tricky with later versions dropping support for Django 1.11, but this version and earlier not supporting Django 3.0. Lets make it `djangorestframework-simplejwt>=4.3.0` for now.

Reply:
This also needs to go inside the [install_requires](https://github.com/lgoodridge/django-uniauth/blob/master/setup.py#L19) section in `setup.py`, since that determines the dependencies for the package.